scripts/generate-roar.sh

```diff
@@ -0,0 +1,88 @@
+#!/bin/bash
+
+if [[ -z "$1" ]]; then
+    echo >&2 "Usage: $0 [ 01 | .. | 25 ] [ gee | roar ]"
+    echo >&2
+    exit
+fi
+
+if [[ ! "$2" =~ ^[0-9][0-9]$ ]]; then
+    echo >&2 "Usage: $0 [ 01 | .. | 25 ]"
```

Comment:
Should include the `[ gee | roar ]`

Reply:
Could potentially combine this with the above guard clause using `if [[ ... ]] || [[ ... ]]`